File: mobile_apps/Android/deliveryman/dropoprovider/src/main/java/com/dropo/provider/models/datamodels/ProviderAnalyticDaily.java
package com.dropo.provider.models.datamodels;

import com.google.gson.annotations.SerializedName;

public class ProviderAnalyticDaily {

    @SerializedName("date_in_server_time")
    private String dateInServerTime;

    @SerializedName("date")
    private String date;

    @SerializedName("rejection_ratio")
    private double rejectionRatio;

    @SerializedName("unique_id")
    private int uniqueId;

    @SerializedName("completed_ratio")
    private double completedRatio;

    @SerializedName("rejected")
    private int rejected;

    @SerializedName("cancellation_ratio")
    private double cancellationRatio;

    @SerializedName("total_active_job_time")
    private long totalActiveJobTime = 0;

    @SerializedName("accepted")
    private int accepted;

    @SerializedName("received")
    private int received;

    @SerializedName("completed")
    private int completed;

    @SerializedName("not_answered")
    private int notAnswered;

    @SerializedName("total_online_time")
    private long totalOnlineTime = 0;

    @SerializedName("working_hours")
    private int workingHours;

    @SerializedName("provider_id")
    private String providerId;

    @SerializedName("cancelled")
    private int cancelled;

    @SerializedName("_id")
    private String id;

    @SerializedName("tag")
    private String tag;

    @SerializedName("acception_ratio")
    private double acceptionRatio;

    public String getDateInServerTime() {
        return dateInServerTime;
    }

    public void setDateInServerTime(String dateInServerTime) {
        this.dateInServerTime = dateInServerTime;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public double getRejectionRatio() {
        return rejectionRatio;
    }

    public void setRejectionRatio(double rejectionRatio) {
        this.rejectionRatio = rejectionRatio;
    }

    public int getUniqueId() {
        return uniqueId;
    }

    public void setUniqueId(int uniqueId) {
        this.uniqueId = uniqueId;
    }

    public double getCompletedRatio() {
        return completedRatio;
    }

    public void setCompletedRatio(double completedRatio) {
        this.completedRatio = completedRatio;
    }

    public int getRejected() {
        return rejected;
    }

    public void setRejected(int rejected) {
        this.rejected = rejected;
    }

    public double getCancellationRatio() {
        return cancellationRatio;
    }

    public void setCancellationRatio(double cancellationRatio) {
        this.cancellationRatio = cancellationRatio;
    }

    public long getTotalActiveJobTime() {
        return totalActiveJobTime;
    }

    public void setTotalActiveJobTime(long totalActiveJobTime) {
        this.totalActiveJobTime = totalActiveJobTime;
    }

    public int getAccepted() {
        return accepted;
    }

    public void setAccepted(int accepted) {
        this.accepted = accepted;
    }

    public int getReceived() {
        return received;
    }

    public void setReceived(int received) {
        this.received = received;
    }

    public int getCompleted() {
        return completed;
    }

    public void setCompleted(int completed) {
        this.completed = completed;
    }

    public int getNotAnswered() {
        return notAnswered;
    }

    public void setNotAnswered(int notAnswered) {
        this.notAnswered = notAnswered;
    }

    public long getTotalOnlineTime() {
        return totalOnlineTime;
    }

    public void setTotalOnlineTime(long totalOnlineTime) {
        this.totalOnlineTime = totalOnlineTime;
    }

    public int getWorkingHours() {
        return workingHours;
    }

    public void setWorkingHours(int workingHours) {
        this.workingHours = workingHours;
    }

    public String getProviderId() {
        return providerId;
    }

    public void setProviderId(String providerId) {
        this.providerId = providerId;
    }

    public int getCancelled() {
        return cancelled;
    }

    public void setCancelled(int cancelled) {
        this.cancelled = cancelled;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getTag() {
        return tag;
    }

    public void setTag(String tag) {
        this.tag = tag;
    }

    public double getAcceptionRatio() {
        return acceptionRatio;
    }

    public void setAcceptionRatio(double acceptionRatio) {
        this.acceptionRatio = acceptionRatio;
    }


}

File: mobile_apps/IOS/deliveryman/Edelivery Provider/Models/DataModel/Earning Response/ProviderAnalyticDaily.swift
//
//	ProviderAnalyticDaily.swift
//
//	Create by Elluminati on 28/6/2017
//	Copyright © 2017 Elluminati. All rights reserved.

import Foundation
class ProviderAnalyticDaily {
    var v : Int!
	var id : String!
	var accepted : Int!
	var acceptionRatio : Double!
	var cancellationRatio : Double!
	var cancelled : Int!
	var completed : Int!
	var completedRatio : Double!
	var date : String!
	var dateInServerTime : String!
	var notAnswered : Int!
	var providerId : String!
	var received : Int!
	var rejected : Int!
	var rejectionRatio : Double!
	var tag : String!
	var totalActiveJobTime : Int64!
	var totalOnlineTime : Int64!
	var uniqueId : Int!
	var workingHours : Int!

	/**
	 * Instantiate the instance using the passed dictionary values to set the properties values
	 */
	init(fromDictionary dictionary: [String:Any]){
		v = (dictionary["__v"] as? Int) ?? 0
		id = (dictionary["_id"] as? String) ?? ""
		accepted = (dictionary["accepted"] as? Int) ?? 0
		acceptionRatio = (dictionary["acception_ratio"] as? Double) ?? 0.0
		cancellationRatio = (dictionary["cancellation_ratio"] as? Double) ?? 0.0
		cancelled = (dictionary["cancelled"] as? Int) ?? 0
		completed = (dictionary["completed"] as? Int) ?? 0
		completedRatio = (dictionary["completed_ratio"] as? Double) ?? 0.0
		date = (dictionary["date"] as? String) ?? ""
		dateInServerTime = (dictionary["date_in_server_time"] as? String) ?? ""
		notAnswered = (dictionary["not_answered"] as? Int) ?? 0
		providerId = (dictionary["provider_id"] as? String) ?? ""
		received = (dictionary["received"] as? Int) ?? 0
		rejected = (dictionary["rejected"] as? Int) ?? 0
		rejectionRatio = (dictionary["rejection_ratio"] as? Double) ?? 0.0
		tag = (dictionary["tag"] as? String) ?? ""
		totalActiveJobTime = (dictionary["total_active_job_time"] as? Int64) ?? 0
		totalOnlineTime = (dictionary["total_online_time"] as? Int64) ?? 0
		uniqueId = (dictionary["unique_id"] as? Int) ?? 0
		workingHours = (dictionary["working_hours"] as? Int) ?? 0
        
	}

}
